minigotchi-ESP32/config.cpp

```diff
@@ -3,6 +3,7 @@
 */
 
 #include "config.h"
+#include <TFT_eSPI.h>
```

Comment:
this would belong in `display.cpp`, `config.cpp` is meant for overall minigotchi configuration.

Reply:
Resolved.